libs/mini-rx-store/src/lib/state.ts

```diff
@@ -1,7 +1,7 @@
 import { BehaviorSubject, Observable } from 'rxjs';
 import { filter } from 'rxjs/operators';
 import { calcNewState, select } from './utils';
```

Comment:
calcNewState exists in the common lib with another name: libs/common/src/lib/calc-next-state.ts